fedimint/Cargo.toml

```diff
@@ -61,3 +61,15 @@ tracing-subscriber = { version = "0.3.1", features = [ "env-filter" ] }
 tracing-opentelemetry = { version = "0.17.2", optional = true}
 threshold_crypto = { git = "https://github.com/fedimint/threshold_crypto" }
 url = { version = "2.2.2", features = ["serde"] }
+
+# setup dependencies
```

Comment:
In terms of security and complexity, it might be best to separate out any UI code/dependencies from the fedimint server itself.  Perhaps we can move this to a separate crate that depends on the fedimint crate?

Reply:
This would be better, but would require moving a lot of stuff to “Core” crate.